### ISSUE TITLE
fleurs benchmark show very high wer in indonesian language

### ISSUE THREAD

**Issue by acul3** [FunAudioLLM/Fun-ASR#17]
Model : FunAudioLLM/Fun-ASR-MLT-Nano-2512 (last update)
Setting :  m.inference(data_in=[torch.tensor(item["audio"]["array"])],language="id", itn=True, **kwargs)
dataset: fleurs id test

Result:
WER raw: 0.5576261038766652 (50%)
WER normalized: 0.47684365781710913 (40%)

it different from what https://funaudiollm.github.io/funasr/ , 6.56(0.06)

Code:
```import subprocess
import sys
import importlib.util
import os

# --- Dependency Check Block ---
required_packages = [
    "datasets", 
    "evaluate", 
    "transformers", 
    "jiwer", 
    "torchcodec"
]
try:
    for package in required_packages:
        # Check if the package is installed
        if importlib.util.find_spec(package) is None:
            print(f"Installing {package}...")
            try:
                # This runs 'pip install <package>' using the current python executable
                subprocess.check_call([sys.executable, "-m", "pip", "install", package])
            except Exception as e:
                print(f"Failed to install {package}: {e}")
except Exception as e:
    print(f"An error occurred during the dependency check: {e}")
# --- Your original imports ---

import os
#install datasets, evaluate, transformer, jiwer,torchcodec if not installed
from datasets import load_dataset
from evaluate import load
from model import FunASRNano
import torch
from transformers.models.whisper.english_normalizer import BasicTextNormalizer


normalizer = BasicTextNormalizer()
fleurs = load_dataset("google/fleurs", "id_id", split="test")
model_dir = "FunAudioLLM/Fun-ASR-MLT-Nano-2512"
m, kwargs = FunASRNano.from_pretrained(model=model_dir, device="cuda:0")
m.eval()

wer_metric = load("wer")

prediction_raws = []
reference_raws = []
predictions_normalize = []
references_normalize = []
for item in fleurs:
    res = m.inference(data_in=[torch.tensor(item["audio"]["array"])],language="id", itn=True, **kwargs)
    text = res[0][0]["text"]
    prediction_raws.append(text)
    reference_raws.append(item["transcription"])
    predictions_normalize.append(normalizer(text))
    references_normalize.append(normalizer(item["transcription"]))
    print(f"REF: {item['transcription']}")
    print(f"HYP: {text}")
    print("-----")
wer_raw = wer_metric.compute(references=reference_raws, predictions=prediction_raws)
wer_normalized = wer_metric.compute(references=references_normalize, predictions=predictions_normalize)
print(f"WER raw: {wer_raw}")
print(f"WER normalized: {wer_normalized}")```

**Comment by acul3** [comment]
[fleurs_id_results.txt](https://github.com/user-attachments/files/24191382/fleurs_id_results.txt)
[fleurs_id_results_normalize.txt](https://github.com/user-attachments/files/24191381/fleurs_id_results_normalize.txt)

**Comment by LauraGPT** [comment]
Maybe something goes wrong, and we would check it sooner.

**Comment by pengzhendong** [comment]
Pull the latest code and model, and try it again.

**Comment by LauraGPT** [comment]
Closing due to inactivity. Please reopen if still relevant with latest version. For usage docs: https://modelscope.github.io/FunASR/tutorial.html
